src/cpip-pack

```diff
@@ -73,7 +73,7 @@ install_compilers()
         fi
 
         # install compilers
-        local pkgs=("$cc_pkg" "$cxx_pkg" "$gfortran_pkg" "$swig_pkg")
+        local pkgs=("$cc_pkg" "$cxx_pkg" "$gfortran_pkg" "$ld_impl")
```

Comment:
Should it not be an addition rather than a replacement?

Reply:
humm, yes, something went wrong on the update. fixing

Reply:
done